apps/omg_watcher/test/omg_watcher/exit_processor/core_test.exs

```diff
@@ -1550,22 +1521,19 @@ defmodule OMG.Watcher.ExitProcessor.CoreTest do
     @tag fixtures: [:alice, :processor_filled, :transactions, :competing_transactions]
     test "handle two competitors, when the younger one already challenged",
          %{alice: alice, processor_filled: processor, transactions: [tx1 | _], competing_transactions: [comp | _]} do
-      txbytes = Transaction.raw_txbytes(tx1)
-      other_txbytes = Transaction.raw_txbytes(comp)
-
-      %{signed_tx: %{sigs: [other_signature, _]}} =
-        other_recovered = OMG.TestHelper.sign_recover!(comp, [alice.priv, alice.priv])
+      txbytes = txbytes(tx1)
+      {comp_txbytes, comp_signature} = {txbytes(comp), sig(comp)}
```

Comment:
welcome to the dark side ;-)

Reply:
do you mean the abbreviations or the tuple magic? :)

If the former - I was also a bit reluctant at first, but the brevity benefits + using a rather repeating (and scope-limited) abbreviations `tx`, `comp`, `sig`... so here I am :joy: